docs/src/screens/docs/sidebar.js

```diff
@@ -80,8 +80,7 @@ class Sidebar extends React.Component {
           <SubContentWrapper>
             {subContent.map(sh => {
               const slug = `#${sh.content
-                .replace('&', '')
-                .replace('.', '')
+                .replace(/[&,:]+/gm, '')
```

Comment:
I combined @boygirl 's earlier fix to one line and added `:` which we have in headings.